all-requirements.txt

```diff
@@ -3,5 +3,9 @@
 # For simpl.rest
 bottle==0.12.8
 
+# For simpl.server
+eventlet==0.17.4
+tornado==4.2
```

Comment:
Yes, absolutely. I'm not even sure separate files is the right approach. We could use a follow-up PR to facilitate that discussion.